### ISSUE TITLE
Standardize on buffalo server?

### ISSUE THREAD

**Issue by arschles** [factorapp/structure#3]
not sure what kind of server to use? maybe `factor build` or whatever generates code should generate a buffalo app...

**Comment by arschles** [comment]
maybe we should standardize on buffalo?

**Comment by arschles** [comment]
If we don't standardize on a buffalo server (maybe even if we do) it would be cool to have some code that can deal with templates and basic web serving needs. there's a crappy start to that [here](https://github.com/factorapp/structure/tree/b9a44de4c80b451ce8580e09330997c868d13ff8/server)

**Comment by ghost** [comment]
:+1: for buffalo server, otherwise we end up reinventing most it

**Comment by arschles** [comment]
Good point. Hoping we can reuse some/most of it as a library. I think it
would be amazing if we didn’t need to execute the Buffalo CLI behind the
scenes because that’s just another prerequisite for developers to install
